src/app/@ansyn/ansyn/app-providers/overlay-source-providers/open-aerial-source-provider.ts

```diff
@@ -0,0 +1,106 @@
+			fetchParams.region = geojsonMultiPolygonToPolygon(fetchParams.region as GeoJSON.MultiPolygon);
+		}
+		let bbox = bboxFromGeoJson(fetchParams.region as GeoJSON.Polygon);
+		// if limit not provided by config - set default value
+		fetchParams.limit = fetchParams.limit ? fetchParams.limit : DEFAULT_OVERLAYS_LIMIT;
+		let baseUrl = this._overlaySourceConfig.baseUrl;
+		// add 1 to limit - so we'll know if provider have more then X overlays
+		let urlWithParams = `${baseUrl}?limit=${fetchParams.limit + 1}&bbox=${bbox[0]}%2C${bbox[1]}%2C${bbox[2]}%2C${bbox[3]}`;
+		return this.http.get<OverlaysOpenAerialFetchData>(urlWithParams)
+			.map(data => {
+				return this.extractArrayData(data.results);
+			})
+			.map((overlays: Overlay[]) => limitArray(overlays, fetchParams.limit, {
+				sortFn: sortByDateDesc,
+				uniqueBy: o => o.id
+			}))
+			.catch((error: Response | any) => {
+				return this.errorHandlerService.httpErrorHandle(error);
+			});
+	}
+
+	getById(id: string, sourceType: string): Observable<Overlay> {
+		let baseUrl = this._overlaySourceConfig.baseUrl;
+		let urlWithParams = `${baseUrl}?_id=${id}&properties.sensor=${sourceType}`;
+		return this.http.get<OverlaysOpenAerialFetchData>(urlWithParams)
+			.map(data => {
+				return this.extractData(data.results);
+			})
+			.catch((error: Response | any) => {
+				return this.errorHandlerService.httpErrorHandle(error);
+			});
+	}
+
+	getStartDateViaLimitFacets(params: { facets; limit; region }): Observable<StartAndEndDate> {
+		return Observable.empty();
+	}
+
+	getStartAndEndDateViaRangeFacets(params: { facets; limitBefore; limitAfter; date; region }): Observable<any> {
+		return Observable.empty();
+	}
+
+	private extractArrayData(overlays: Array<any>): Array<Overlay> {
+		return overlays ? overlays.map((element) => {
+			return this.parseData(element);
+		}) : [];
+	}
+
+	private extractData(overlays: Array<any>): Overlay {
+		return this.parseData(overlays[0]);
+	}
+
+	protected parseData(openAerialElement: any): Overlay {
+		let overlay: Overlay = new Overlay();
+		const footprint: any = wellknown.parse(openAerialElement.footprint);
+		overlay.id = openAerialElement._id;
+		overlay.footprint = footprint.geometry ? footprint.geometry : footprint;
+		overlay.sensorType = openAerialElement.properties.sensor;
+		overlay.sensorName = openAerialElement.platform;
+		overlay.channel = openAerialElement._v; // not sure what is channel for
+		overlay.bestResolution = openAerialElement.gsd;
+		overlay.name = openAerialElement.title;
+		overlay.imageUrl = openAerialElement.properties.tms ? openAerialElement.properties.tms : openAerialElement.properties.wmts; // not always just tms
+		overlay.thumbnailUrl = openAerialElement.properties.thumbnail;
+		overlay.date = new Date(openAerialElement.acquisition_end);
+		overlay.photoTime = openAerialElement.acquisition_end;
+		overlay.azimuth = toRadians(180 - 91.2); // how? just a number i found in an idaho request
```

Comment:
its not a const value `toRadians(180 - idahoElement.properties.satAzimuth)` 
please search about `openAerialElement` data model, and find the azimuth value